data/frame_type.go

```diff
@@ -41,6 +41,20 @@ const (
 	// This structure is typically part of a list of frames with the same structure
 	FrameTypeTimeSeriesMany = "timeseries-many"
 
+	// FrameTypeLabeledTimeValues has at least three fields:
+	// field[0]:
+	//  * type: string field
+	//  * name: "labels"
```

Comment:
"labels" or "Labels"? i personally like lowercase-names more, but on the other hand, we seem to default to capital-case with these, like "Time" and "Value" in https://github.com/grafana/grafana-plugin-sdk-go/blob/main/data/constants.go or in https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/types/dataFrame.ts#L224-L225 ..

so maybe "Labels" is more consistent?